packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -14,19 +14,27 @@ const chatMessageStyles: ComponentSlotStylesInput<
 > = {
   root: ({ props: p, variables: v }): ICSSInJSStyle => ({
     display: 'inline-block',
+    position: 'relative',
+
+    marginLeft: p.mine ? pxToRem(140) : 0,
```

Comment:
Shouldn't this be `100px`? I think we should calculate with rem being `10px` when converting this values.